modulles/news.modules.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../db/connection");
+
+exports.receiveAllTopics = (req, res) => {
+  return db.query("SELECT * FROM topics;");
```

Comment:
This is up to you, but I'd try to keep your controller as streamline as possible and return rows here rather than the controller.